plugins/forward/alerta_forward.py

```diff
@@ -0,0 +1,28 @@
+import logging
+
+from alerta.plugins import PluginBase
+from alertaclient.api import Client
+
+LOG = logging.getLogger('alerta.plugins.forward')
+
+FORWARD_URL = app.config.get('FORWARD_URL')
+FORWARD_API = app.config.get('FORWARD_API')
```

Comment:
Can we call the variable `FORWARD_API_KEY` to be consistent with other plugins? And we should allow them to be set from the shell environment as other plugins do -- especially as you say this is possible in the README.